masschallenge/semantic-ui-theme
# [AC-7557] Cancellation workflow for office hours

## [AC-7557](https://masschallenge.atlassian.net/browse/AC-7557)

**Changes Introduced:**
- Add CSS overrides for office hour cancellation workflow

**Testing**
- checkout to `AC-7557` on `accelerate`, `semantic-ui-theme` and `impact-api`
- run servers ( accelerate and impact-api)
- On semantic-ui-theme:
    - run `gulp build`
    - run `yarn link`
- On accelerate
    - run `yarn link "semantic-ui-theme"`
    - run `yarn start`

- Visit [http://localhost:8181/newofficehour](http://localhost:8181/newofficehour)
For test purposes you can set `office_hour_id` and `mentor_id (office hour holder id)` using URL query. i.e `http://localhost:8181/newofficehour?office_hour_id=1&mentor_id=2`
- Verify that the cancellation workflow meets the definition of done according to [AC-7557](https://masschallenge.atlassian.net/browse/AC-7557)

- Allowed users: 
    - Staff users - Can cancel session/reservation on behalf of mentor/finalist
    - Office hour session owner(mentor) - can cancel their unreserved office hour sessions
    - Finalist- can cancel their reserved office hour sessions


**Sibling PRs**
- [Impact api PR](https://github.com/masschallenge/impact-api/pull/414)
- [Accelerate PR](https://github.com/masschallenge/accelerate/pull/2614)
